src/cmd/list.go

```diff
@@ -211,12 +216,29 @@ func listOutput(images, containers []map[string]interface{}) {
 			"STATUS",
 			"IMAGE NAME")
 
+		isPodmanV2 := podman.CheckVersion("2.0.0")
+		var idKey, createdKey, statusKey string
+		if isPodmanV2 {
+			idKey = "Id"
+			createdKey = "CreatedAt"
+			statusKey = "State"
+		} else {
+			idKey = "ID"
+			createdKey = "Created"
+			statusKey = "Status"
+		}
+
+		var id, name, created, status, imageName interface{}
 		for _, container := range containers {
-			id := container["ID"].(string)[:12]
-			name := container["Names"].(string)
-			created := container["Created"].(string)
-			status := container["Status"].(string)
-			imageName := container["Image"].(string)
+			id = utils.ShortID(container[idKey].(string))
```

Comment:
Nitpick: this bit should be split out into a separate commit since it's unrelated to Podman 2.0.

See: https://github.com/containers/toolbox/pull/479

Reply:
Ah, right. Thanks for the catch.